website/docs/architecture/chaos-execution-plane.md

```diff
@@ -0,0 +1,42 @@
+---
+id: chaos-execution-plane
+title: Chaos Execution Plane
+sidebar_label: Chaos Execution Plane
+---
+
+---
+
+<img src={require("../assets/execution-plane.png").default} alt="Chaos Execution Plane" />
+
+Chaos Execution Plane contains the components responsible for orchestrating the chaos injection in the target resources. They get installed in either an external target cluster if an external agent is being used or in the host cluster containing the control plane if a self-agent is being used. It can be further segregated into Litmus Agent Infrastructure components and Litmus Backend Execution Infrastructure components. 
+
+Litmus Agent Infrastructure components help facilitate the chaos injection, manage chaos observability, and enable chaos automation for target resources. These components include:
+
+1. **Argo Workflow Controller:** Used for the creation of Chaos Workflows for Litmus using the Chaos Workflow CR.
+
+2. **Subscriber:** Serves as the link between the Chaos Execution Plane and the Control Plane. It has a few distinct responsibilities including the creation of a Chaos Workflow CR from a Chaos Workflow template, watching for Chaos Workflow events during its execution, and sending the chaos workflow result to the Control Plane. 
+
+3. **Event Tracker:** An optional component that is capable of triggering automated chaos workflow runs based on a set of defined conditions. It is managed by the Backend GitOps Controller, and can only be used if GitOps is enabled.
```

Comment:
The event tracker is by itself a controller & works with eventTrackerPolicy CR - we may want to relook at the 'backend GitOps controller' - @rajdas98 @gdsoumya